src/modelversionstatus.hpp

```diff
@@ -118,6 +118,10 @@ class ModelVersionStatus {
         return ovms::ModelVersionState::UNLOADING <= this->state;
     }
 
+    bool isFailedLoading() const {
+        return this->state == ovms::ModelVersionState::LOADING && this->errorCode == ovms::ModelVersionStatusErrorCode::UNKNOWN;
```

Comment:
In future, we could report why loading has failed using errorCode which is then nicely translated to proto.